### DISCUSSION TITLE
yiwen-blog-website/docs/native/android/rn_qs/

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#258] [Announcements]
# yiwen-blog-website/docs/native/android/rn_qs/

command node path问题

https://hao-yiwen.github.io/yiwen-blog-website/docs/native/android/rn_qs/

**Comment by Hao-yiwen** [comment]
今天再解决问题的时候发现只要和node相关的，这个命令大多数都能解决问题。问题根因感觉还是在gradle中。
